include/cantera/oneD/StFlow.h

```diff
@@ -19,12 +19,15 @@ namespace Cantera
 //------------------------------------------
 
 // Offsets of solution components in the solution array.
-const size_t c_offset_U = 0; // axial velocity
-const size_t c_offset_V = 1; // strain rate
-const size_t c_offset_T = 2; // temperature
-const size_t c_offset_L = 3; // (1/r)dP/dr
-const size_t c_offset_E = 4; // electric poisson's equation
-const size_t c_offset_Y = 5; // mass fractions
+enum offset
```

Comment:
I think the comment above should be a Doxygen comment (starting with `//!`) to get this to show up in the Doxygen docs. Also, you should build the docs and check what happens with the comments on the individual members. I expect that you'll want to start those with `//!<` to get them to show up as referencing the correct constant.

Reply:
@BangShiuh … could you address so this can be merged?

Reply:
I got
```
<html>
<body>
<!--StartFragment-->

Enumerator
--
c_offset1D_U | axial velocity
c_offset1D_V | strain rate
c_offset1D_T | temperature
c_offset1D_L | (1/r)dP/dr
c_offset1D_E | electric poisson's equation
c_offset1D_Y | mass fractions

<!--EndFragment-->
</body>
</html>
```